SR2025/2025-01-15-check-in.md

```diff
@@ -0,0 +1,27 @@
+---
+to: SR2025 teams
+subject: Student Robotics 2025 {TLA} January Check-in
+---
+
+Hi {PrimaryGivenName},
+
+Happy New Year!
+
+Just wanted to check in on how your team is doing?
+
+It would be great to hear what your team is currently working on and if there's
+anything you'd like a hand with.
+
+A couple of quick reminders for the next few weeks:
+
+* if you'd like to come to the [Cambridge Tech Day][cambridge-tech-day] on
+  Saturday please [let us know][tech-day-signup] this week
```

Comment:
This sounds like we're not going to send this until the week of the tech day. This is really too close to be useful to them.

Reply:
Why not? This isn't the primary comms about the tech day. Do you think we should drop this?

Reply:
```suggestion
* if you'd like to come to the [Cambridge Tech Day][cambridge-tech-day] on
  Saturday 25th January, please [let us know][tech-day-signup] by the 17th.
```

Reply:
Please see other discussion about details vs brevity and consider the intended (personal, colloquial) tone of this email. Given those factors these additional details are deliberately being omitted.

Reply:
If we aren't going to make the tech day a focus of this email then we need to send out an email advertising the tech day this week.

Reply:
```suggestion
* if you'd like to come to the [Cambridge Tech Day][cambridge-tech-day] next
  Saturday please [let us know][tech-day-signup] this week
```